packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -516,6 +516,9 @@ class TimeInput extends React.Component {
       ...customProps
     } = this.props;
 
+    const instanceHoursAttrs = Object.assign({}, hourAttributes);
```

Comment:
I'm curious, does creating a new hours and minutes attributes object change how the id are attached to the rendered component?

Reply:
Yeah. What I found is that all instances of the time input share the same default prop object as opposed to having their own instance. If I rely on the default hourAttributes prop for a time input and it get's modified (like setting the id), that same id carries over to the next time input.

Reply:
oh, that is interesting. who knew 🤷‍♀️